### PR TITLE
Feature/wasm param changes

### PR DESCRIPTION
**Pull request by ouiliame**
Adds WASM module's parameter change options to ParamChanges

### CODE REVIEW

File: src/core/wasm/msgs/MsgMigrateContract.ts [hunk 1]
```diff
@@ -8,13 +8,13 @@ export class MsgMigrateContract extends JSONSerializable<
...
    * @param contract contract address to be migrated from
    * @param new_code_id reference to the new code on the blockchain
-   * @param migrate_msg message to configure the migrate state of the contract
+   * @param migrate_msg JSON message to configure the migrate state of the contract
    */
   constructor(
     public owner: AccAddress,
     public contract: AccAddress,
     public new_code_id: number,
-    public migrate_msg: string
```
Comment (hanjukim) [inline src/core/wasm/msgs/MsgMigrateContract.ts:17]: why not use object instead of any

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.4.5",
+  "version": "0.4.8",
```
Comment (hanjukim) [inline package.json:2]: why 0.4.5 > 0.4.8?

Comment (ouiliame) [reply package.json:2]: agh sorry made edits from wrong branch

Comment (ouiliame) [reply package.json:2]: current ver is 0.4.7 (with Coins constructor)

File: src/core/wasm/msgs/MsgMigrateContract.ts (hunk 1 above)
Comment (ouiliame) [reply src/core/wasm/msgs/MsgMigrateContract.ts:17]: good point

File: package.json (hunk 2 above)
Comment (hanjukim) [reply package.json:2]: got it

**Pull request merged**